uint8 control_mode
uint8 enabled
float32 value